ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -2159,6 +2169,18 @@ FCU_DFSS_DFRMM_CHWBZC:
   - DFRMM
   - CHWBZC
 
+FCU_SS_DFSS_ZTC:
+  guid: "d2b3f37b-d69f-4631-b8d2-d71243821e1c"
+  description: "FCU with discharge fan and zone temperature control"
+  is_canonical: true
+  implements:
+  - FCU
+  - SS
```

Comment:
What is SS used for?

Reply:
it is used to get the control mode but maybe it better to change to DFVSC instead

Reply:
I have changed to DFVSC